### PR TITLE
front/back -> stencilFront/stencilBack, stencilFailOp -> failOp

### LINKED ISSUE #159: DepthStencilStateDescriptor doesn't specify that "back" and "front" are for stencil
It's weird that `depthStencilStateDescriptor.back.compare` doesn't specify that it's for stencil. It should probably be `.stencilBack.compare` (or maybe `.back.stencilCompare`).

current:
- WebGPUDepthStencilStateDescriptor
  - depthWriteEnabled
  - depthCompare
  - front
    - compare
    - stencilFailOp
    - depthFailOp
    - passOp
  - back
    - compare
    - stencilFailOp
    - depthFailOp
    - passOp
  - stencilReadMask
  - stencilWriteMask

proposal: `front` -> `stencilFront`, `back` -> `stencilBack`.

An alternative would be to split even further, but probably goes too far:
- WebGPUDepthStencilStateDescriptor
  - depth
    - writeEnabled
    - compare
  - stencil
    - front
      - compare
      - stencilFailOp
      - depthFailOp
      - passOp
    - back
      - compare
      - stencilFailOp
      - depthFailOp
      - passOp
    - readMask
    - writeMask

### PR DESCRIPTION
**Pull request by kainino0x**
fixes #159

### CODE REVIEW

File: design/sketch.webidl [hunk 1]
```diff
@@ -629,7 +629,7 @@ interface WebGPUQueue {
     void wait(WebGPUFence fence, u64 valueToWait);
 };
 
-// SwapChain / RenderingContext
+// SwapChain / RenderingContextf
```
Comment (Kangz) [inline design/sketch.webidl]: nit: extra f added

**Comment by kvark** [comment]
Sorry to be that guy, but... please rebase and squash the commits like "typo"

**Comment by kainino0x** [comment]
Np, I did this via web interface but I'm already pulling it down to fix the commits locally.

BTW, I'm pretty sure "Squash and merge" can handle the merge commit too.

**Comment by kainino0x** [comment]
Thanks, going ahead with merge.

**Pull request merged**